src/logs/logs.proto

```diff
@@ -48,9 +48,14 @@ message PodContainer {
 
 // LogLevel selects a log level. Pachyderm services only have DEBUG, INFO, and ERROR.
 enum LogLevel {
-  LOG_LEVEL_DEBUG = 0;
-  LOG_LEVEL_INFO = 1;
-  LOG_LEVEL_ERROR = 2;
+  // Use default log level filtering.
+  LOG_LEVEL_UNSET = 0;
+  // Include DEBUG, INFO, and ERROR logs.
+  LOG_LEVEL_DEBUG = 1;
+  // Include INFO and ERROR logs.
+  LOG_LEVEL_INFO = 2;
+  // Include only ERROR logs.
+  LOG_LEVEL_ERROR = 3;
```

Comment:
This is probably the best set of enum members, _but_ it does change released code. I think this is still probably okay, since nothing (to my knowledge) has been using this yet.

Reply:
Nothing we can do here.  Someone should have caught on the initial checkin that the 0 value wasn't UNKNOWN/UNSET/etc.